src/main/java/uk/gov/justice/digital/job/cdc/TableStreamingQuery.java

```diff
@@ -27,8 +27,6 @@ public class TableStreamingQuery {
     private final JobArguments arguments;
     private final S3DataProvider s3DataProvider;
     private final CdcBatchProcessor batchProcessor;
-    private final String inputSchemaName;
```

Comment:
Now that we don't use both nomis and oms_owner in different places the distinction between these Strings and the source and table inside source reference is moot - so I have removed it.